### PR TITLE
Rightnav Tweaks (drop zapcon)

### PR DESCRIPTION
**Pull request by kingthorin**

### CODE REVIEW

File: info.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
-### Project Classification
+### Project Information
 
-* <i class="fas fa-flag" style="color:#2ADA08;"></i> Flagship Project
-* <i class="fas fa-hammer" style="color:#233e81;"></i> Breaker
-* <i class="fas fa-toolbox" style="color:#233e81;"></i> Builder
+* <i class="fas fa-flag" style="font-size: 1.3em; color:#38a047;"></i>
+  <span style="font-size: 1.3em;">Flagship Project</span>
+
+### Classification
```
Comment (thc202) [inline info.md:6]: `####`? Seems to look better (e.g. https://owasp.org/www-project-juice-shop/)

File: info.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 
 * <i class="fas fa-tools" style="color:#233e81;"></i> Tool
 
+### Audience
```
Comment (thc202) [inline info.md:10]: Same here.

File: info.md (hunk 1 above)
Comment (kingthorin) [reply info.md:6]: Sure, I’ll tweak that.

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]
Thanks!

**Pull request merged**